components/esp_lvgl_port/src/lvgl9/esp_lvgl_port_disp.c

```diff
@@ -319,4 +456,20 @@ static void lvgl_port_disp_size_update_callback(lv_event_t *e)
         }
         break;
     }
+
+    /* Wake LVGL task, if needed */
+    lvgl_port_task_wake(LVGL_PORT_EVENT_DISPLAY);
+}
+
+static void lvgl_port_disp_size_update_callback(lv_event_t *e)
+{
+    assert(e);
+    lvgl_port_display_ctx_t *disp_ctx = (lvgl_port_display_ctx_t *)e->user_data;
+    lvgl_port_disp_rotation_update(disp_ctx);
+}
+
+static void lvgl_port_display_invalidate_callback(lv_event_t *e)
+{
+    /* Wake LVGL task, if needed */
+    lvgl_port_task_wake(LVGL_PORT_EVENT_DISPLAY);
```

Comment:
https://github.com/lvgl/lvgl/blob/addc3f0c7327f0945d560f06f6cc2487a223a0ef/src/drivers/nuttx/lv_nuttx_lcd.c#L119C13-L119C23
	
In this cb, can we add a size readjustment function, similar to the implementation of the above link.
Due to some LCD rendering sizes must be 4-byte aligned.

Reply:
This is hard to add now. It is hard to test for me, if I haven't got specific HW. I can add it into my notes and add this feature later.

Reply:
> This is hard to add now. It is hard to test for me, if I haven't got specific HW. I can add it into my notes and add this feature later.

ok, we add it next step, and I'll prepare one pcs for you later.

Reply:
Thank you!